pkg/controller/rollout/rollout_status.go

```diff
@@ -122,6 +137,11 @@ func (r *RolloutReconciler) calculateRolloutStatus(rollout *v1alpha1.Rollout) (r
 			}
 			newStatus.Message = "workload deployment is completed"
 		}
+	case v1alpha1.RolloutPhaseDisabled:
+		if !rollout.Spec.Disabled {
+			newStatus.Phase = v1alpha1.RolloutPhaseHealthy
```

Comment:
newStatus.Phase = v1alpha1.RolloutPhaseInitial, Because workload may be not found.